route-rs-runtime/src/element/mod.rs

```diff
@@ -9,3 +9,5 @@ pub use self::transform_elements::*;
 
 mod dec_ip_hop;
 pub use self::dec_ip_hop::*;
+
+pub mod utils;
```

Comment:
This seems like the most logical place for network helpers, thoughts?

Reply:
Seems like a decent location, if it needs to be moved later, it isn't a big deal.